src/lib/Helper/ContentHelper.php

```diff
@@ -18,47 +18,57 @@
 use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
 use eZ\Publish\Core\MVC\ConfigResolverInterface;
 use EzSystems\EzRecommendationClient\Value\Parameters;
+use Ibexa\Personalization\Config\Repository\RepositoryConfigResolverInterface;
 use Psr\Log\LoggerInterface;
 
 final class ContentHelper
 {
-    /** @var \eZ\Publish\Api\Repository\SearchService */
-    private $searchService;
+    private const UPDATE_CONTENT_URL_SUFFIX = '%s/api/ezp/v2/ez_recommendation/v1/content/%s/%s%s';
+    private const CONTENT_ID_URL_PREFIX = 'id';
+    private const CONTENT_REMOTE_ID_URL_PREFIX = 'remote-id';
+
+    /** @var \eZ\Publish\Core\MVC\ConfigResolverInterface */
+    private $configResolver;
 
     /** @var \eZ\Publish\API\Repository\ContentService */
     private $contentService;
 
+    /** @var \EzSystems\EzRecommendationClient\Helper\ContentTypeHelper */
+    private $contentTypeHelper;
+
     /** @var \eZ\Publish\API\Repository\LocationService */
     private $locationService;
 
-    /** @var \eZ\Publish\Core\MVC\ConfigResolverInterface */
-    private $configResolver;
+    /** @var \Psr\Log\LoggerInterface */
+    private $logger;
```

Comment:
Looks like `LoggerAwareInterface` / `LoggerAwareTrait` + phpdoc with property declaration (`@property $logger LoggerInterface`) should be used instead.

Reply:
Fixed: https://github.com/ezsystems/ezrecommendation-client/pull/106/commits/dfb3372534f0a0eda284f591fe97e43ec1141b82